CONTRIBUTING.md

```diff
@@ -53,24 +53,32 @@ For other ways to use `tldrl`, such as linting an entire directory, check out (w
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tldrl.md)
 
 ### Token syntax
-User-provided values should use the `{{token}}` syntax in order to allow clients
-to highlight them. 
+User-provided values should use the `{{token}}` syntax
+in order to allow clients to highlight them.
+Use [`snake_case`](https://en.wikipedia.org/wiki/Snake_case)
+for multi-word tokens.
 
-Some examples: 
-- `tar cf {{file}}`
-- `ln -s {{path/to/original/file}} {{path/to/link}}`
-- `mysql {{database_name}}`
-- `unrar x {{compressed.rar}}`
+Keep the following guidelines in mind when writing the tokens:
+
+- If the example is clearer with an actual value rather than a generic placeholder, use the actual value. For example - use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+
+- For any reference to a path or folder, use `{{path/to/<placeholder>}}`. For example - `ln -s {{path/to/file}} {{path/to/symlink}}`. In case of a possible reference both to a file or a folder, use `{{path/to/file_or_folder}}`
+
+- If a command expects the file to be in a particular extension, use it. For example - `unrar x {{compressed.rar}}`. In case needs a generic extension, use `{{.ext}}`, but **only** if it helps to clarify the command. For example, here -
+
+```
+Open all the files of a given extension in the current directory with the associated application:
+open {{*.ext}}
+```
+
+Its clear that `{{.ext}}` helps to clarify the command. But here - `wc -l {{file}}`, using `{{file}}` makes more sense
```

Comment:
Sugestion: "But here - `wc -l {{file}}`, using `{{file}}` makes more sense" --> "But in a command like `wc -l {{file}}`, using `{{file}}` is sufficiently clear."